docs/ossm/README.md

```diff
@@ -6,4 +6,54 @@ This documentation is specific to the OpenShift Service Mesh product and may dif
 
 # Table of Contents
 
-TODO
+- [Running OSSM 2 and OSSM 3 side by side](#running-ossm-2-and-ossm-3-side-by-side)
+
+
+# Running OSSM 2 and OSSM 3 side by side
+This section describes how to run OSSM 2 and OSSM 3 side by side in one cluster without interfering with each other.
+It's not OSSM 2 -> OSSM 3 migration guide.
+
+To better understand steps described here, it's recommended to read OSSM 2 [deployment models](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-deployment-models) and upstream Istio [Multiple Control Planes](https://istio.io/latest/docs/setup/install/multiple-controlplanes/) documentation. This document assumes OSSM 2.6 is already running and the goal is to install OSSM 3.0.
+
+## Cluster-Wide (Single Tenant) mesh deployment model
+### Required OSSM 2 changes
+1. configure discovery selectors in your SMCP CR:
+
+        meshConfig:
+            discoverySelectors:
+            - matchExpressions:
+                - key: maistra.io/member-of
+                    operator: Exists
+        runtime:
+            components:
+            pilot:
+                container:
+                env:
+                    ENABLE_ENHANCED_RESOURCE_SCOPING: 'true'
+
+### OSSM 3 installation
```

Comment:
I think instead of calling this step OSSM Installation, you can this step `procedure.` This matches the current openshift documentation structure

Reply:
Do you mean to have 'OSSM 3 procedure' instead of 'OSSM 3 installation'?  Installation seems clearer to me.

Reply:
Yes